stable/kommander/values.yaml

```diff
@@ -197,6 +197,7 @@ grafana:
 kommander-ui:
   showCost: true
   impersonateUser: true
+  showCD: true
```

Comment:
what is `showCD` for ?

Reply:
It's a feature flag for turning on Continuous Deployment in Kommander.